products/keda.md

```diff
@@ -20,16 +20,23 @@ identifiers:
 
 # Upcoming release date can be found at https://github.com/kedacore/keda/blob/main/ROADMAP.md.
 releases:
+-   releaseCycle: "2.15"
+    releaseDate: 2024-08-01
+    eol: 2025-01-30 # releaseDate(2.17)
```

Comment:
```suggestion
    eol: 2025-01-30 # estimated releaseDate(2.17)
```

Still and estimation according to https://github.com/kedacore/keda/blob/main/ROADMAP.md.